### PR TITLE
Merging dev -> main

### PR DESCRIPTION
**Pull request by anirbanshaw24**
*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*

Issue #

## Changes description

*Clearly and concisely describe what's in this pull request. Include screenshots, if necessary.*

### CODE REVIEW

File: .github/workflows/pkgdown.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches:
+      - dev # This needs to be updated after moving to main branch
```
Comment (jakubnowicki) [inline .github/workflows/pkgdown.yml:6]:
```suggestion
      - main
```

File: .github/workflows/r.yml [hunk 2]
```diff
@@ -42,6 +43,8 @@ jobs:
         if: always()
         shell: Rscript {0}
         run: |
+          remotes::install_cran("devtools")
+          devtools::load_all()
```
Comment (jakubnowicki) [inline .github/workflows/r.yml:47]: Those 2 lines break the `lintr` check (). Running `lintr::lint_package()` will throw errors. Please remove (check e.g. [this](https://github.com/Appsilon/shiny.i18n/blob/main/.github/workflows/main.yml#L43) for reference).

File: DESCRIPTION [hunk 3]
```diff
@@ -1,24 +1,43 @@
...
+    lintr,
+    spelling,
+    config,
+    BiocManager,
+    optparse,
+    BiocCheck,
+    knitr,
+    rmarkdown,
+    sessioninfo,
+    testthat (>= 3.0.0)
```
Comment (jakubnowicki) [inline DESCRIPTION:41]: Please arrange them in alphabetical order.

File: .Rbuildignore [hunk 4]
```diff
@@ -3,6 +3,7 @@
 ^shiny\.gosling\.Rproj$
 ^\.Rproj\.user$
 ^js$
+^LICENSE\.md$
```
Comment (jakubnowicki) [inline .Rbuildignore:6]: There is no license file, please remove.

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
RCMD check throws warnings, they need to be resolved.
Also, please check my comments.

File: .github/workflows/r.yml (hunk 2 above)
Comment (anirbanshaw24) [reply .github/workflows/r.yml:47]: This was added as some functions seemed to be not loaded despite being present. [Errors here](https://github.com/Appsilon/shiny.gosling/actions/runs/3748765116/jobs/6366478083)

Comment (jakubnowicki) [reply .github/workflows/r.yml:47]: Please change this to follow the simpler pattern that is used in our other packages:
https://github.com/Appsilon/shiny.i18n/blob/main/.github/workflows/main.yml#L35

**Review by jakubnowicki — APPROVED** [review]
LGTM

**Pull request merged**